README.md

```diff
@@ -38,6 +38,8 @@ Visit [RBAC permissions for running the tasks](docs/tasks-rbac-permissions.md) i
 
 ## Usage
 
+#### copy-template, modify-vm-template and create-vm-from-template tasks are deprecated and will be removed in future versions.
```

Comment:
Can we name a replacement yet? Or what's the plan going forward?

Reply:
The plan is to deprecate template based tasks and replace them with create-vm-from-manifest with few enhancements related to instance types

Reply:
Can we add this to the readme to give more context?